oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -426,6 +428,20 @@ private void checkIfStopped() throws CommitFailedException {
 
     @Override
     public synchronized void run() {
+        boolean isIndexWithLanePresent = false;
+        NodeState oakIndexNode = store.getRoot().getChildNode("oak:index");
+        Iterator<? extends ChildNodeEntry> indexNodeIterator = oakIndexNode.getChildNodeEntries().iterator();
+        while (indexNodeIterator.hasNext()){
+            ChildNodeEntry  childNodeEntry = indexNodeIterator.next();
+            if (childNodeEntry.getNodeState().getProperty("async") != null){
```

Comment:
Don't you need to check if the async property (which is possibly multi-valued, but might be just a string) is either equals "name", or contains "name"?

Reply:
done